provider/go.mod

```diff
@@ -1,12 +1,211 @@
-module github.com/pulumi/pulumi-xyz/provider
+module github.com/pulumiverse/pulumi-statuscake/provider
 
 go 1.18
 
 replace (
-	github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/pulumi/terraform-plugin-sdk/v2 upstream-v2.21.0
+	github.com/hashicorp/go-getter v1.5.0 => github.com/hashicorp/go-getter v1.4.0
+	github.com/hashicorp/terraform-plugin-log => github.com/hashicorp/terraform-plugin-log v0.5.0
+	github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/pulumi/terraform-plugin-sdk/v2 v2.0.0-20220505215311-795430389fa7
+	github.com/pulumi/pulumi/pkg/v3 => github.com/pulumi/pulumi/pkg/v3 v3.34.1
 )
 
 require (
-	github.com/pulumi/pulumi-terraform-bridge/v3 v3.28.1
-	github.com/pulumi/pulumi/sdk/v3 v3.38.0
+	github.com/jgautheron/terraform-provider-statuscake/v2 v2.1.0
```

Comment:
Can you explain why the need for the fork? What was not working when using the upstream https://github.com/StatusCakeDev/terraform-provider-statuscake repo?

Reply:
Unfortunately the official provider has the `provider` hidden under `internal/provider` which makes it unaccessible. My fork merely drops the `internal` package and exposes the `provider` directly.
I plan to send a PR to the official provider and explain why it'd be beneficial to expose the provider.

Reply:
Oh beautiful, will change it immediately 👍